src/vs/editor/contrib/quickAccess/browser/gotoLineQuickAccess.ts

```diff
@@ -133,6 +188,12 @@ export abstract class AbstractGotoLineQuickAccessProvider extends AbstractEditor
 
 		// Location invalid: show generic label
 		const position = editor.getPosition() || { lineNumber: 1, column: 1 };
+
+		// When in offset mode, prompt for an offset.
+		if (inOffsetMode) {
+			return localize('gotoLineOffsetLabel', "Current Line: {0}, Character: {1}. Type a character offset to navigate to.", position.lineNumber, position.column);
+		}
+
 		const lineCount = this.lineCount(editor);
 		if (lineCount > 1) {
 			return localize('gotoLineLabelEmptyWithLimit', "Current Line: {0}, Character: {1}. Type a line number between 1 and {2} to navigate to.", position.lineNumber, position.column, lineCount);
```

Comment:
<img width="667" height="176" alt="Image" src="https://github.com/user-attachments/assets/d93860ac-1213-4c2a-a8d3-3612ab3e7ae9" />

I wish this included text around navigating to the offset. I wonder if a prompt makes sense for this. This can be done later though